plugins/design-system/components/component-example/CodeExampleFooter.js

```diff
@@ -0,0 +1,73 @@
+import classNames from "classnames/dedupe";
+import React, { Component } from "react";
+
+const defaultClasses = {
```

Comment:
It's a decision it seems I have to make often when using React. I think it is okay in this case.